packages/web/app/pages/[organizationId]/[projectId]/[targetId]/laboratory.tsx

```diff
@@ -869,7 +864,7 @@ function LaboratoryPageContent() {
               </>
             ),
           }}
-          showPersistHeadersSettings={false}
+          showPersistHeadersSettings
```

Comment:
I figured that always persisting headers can be a security concern, but we can allow turning in on in the settings.

Reply:
Yes, I think this is why we originally disabled it by default. Using the settings to control that is a good decision 👍